src/asciidoc/docs/guides-general/build-custom-controls.adoc

```diff
@@ -0,0 +1,210 @@
+[.callout2]#Creates an instance of `CustomVcc`.# +
+[.callout3]#Starts event listeners to listen for updates.# +
+[.callout4]#After the customer control loads, registers it to trigger events from the parent editor.# +
+[.callout5]#When a user changes the value, updates the custom control.# +
+[.callout6]#Saves the file.# +
+[.callout7]#For testing. Displays the value returned by the custom control.#
+
+To customize the forked code for your custom control:
+
+. Modify any of the default settings and parameters.
+. Add code required by your custom control, such as:
++
+* Elements in the `return` section that users can interact with.
+* Commands in the `finish` section to perform additional processing on the captured input.
+. Clean up unneeded code.
+. When done, publish your new custom control app.
+
+Learn more: <<customvcc-reference#>>
+
+==== Provide a link to your custom control
+
+Any app that uses your custom control must call `Koji.ui.capture.custom` and provide a `koji-vccs.com` subdomain or URL that points to the custom control.
+
+* You can determine the assigned URL of the custom control.
++
+.Learn how ...
+[%collapsible]
+====
+include::{includespath}/steps-codeeditor.adoc[tag=go-to-project-details-base]
+include::{includespath}/steps-codeeditor.adoc[tag=view-published-koji]
+. Note the URL of the displayed page.
+====
+
+* You can get your own subdomain of `koji-vccs.com`.
++
+.Learn how ...
+[%collapsible]
+====
+include::{includespath}/steps-domains.adoc[tag=open-published-custom-control]
++
+include::{includespath}/steps-domains.adoc[tag=open-published-how]
+
+include::{includespath}/steps-domains.adoc[tag=menu-app-custom-domain,click-add-new-domain,select-subdomain]
+
+include::{includespath}/steps-domains.adoc[tag=create-koji-vccs-com-subdomain]
++
+This name will be passed as the `name` parameter to  `Koji.ui.capture.custom`.
+
+include::{includespath}/steps-domains.adoc[tag=click-register]
+====
+
+* You can also <<custom-domains#_using_a_domain_you_own,use a domain you own>>.
+
+==== Test your custom control
+
+. Create a standalone app.
++
+You can use any scaffold for a standalone Koji app, such as the  https://withkoji.com/apps/1065866f-c4b2-4950-9aaa-20faa8e2ac8b[Basic React/Express Scaffold].
+. Add an element, such as a button that triggers the call to the custom control.
+. To call the custom control and capture its value, use `Koji.ui.capture.custom`.
++
+* Using the custom control's `koji-vccs.com` subdomain:
++
+[source,javascript]
+----
+Koji.ui.capture.custom({ name: 'mySubdomain' });
+----
++
+* Using any URL that points to the custom control:
++
+[source,javascript]
+----
+Koji.ui.capture.custom({ url: 'https://example.com/url-to-my-koji-custom-control' });
+----
+
+Example: https://withkoji.com/@eltesorokoji/nb9e[Tester App for Custom Control]
+
+== Related resources
+
+* <<customvcc-reference#d>>
+* <<core-frontend-ui-capture#>>
```

Comment:
Due to a bug in our custom parser, you must provide link text for this link. See DC-593 for more info.

Reply:
* Added link texts for both.
* The "d" after the "#" was a typo. Fixed.